packages/fxa-settings/src/pages/InlineRecoveryKeySetup/index.tsx

```diff
@@ -59,19 +60,15 @@ export const InlineRecoveryKeySetup = ({
       case 2:
         return (
           <>
-            <Banner type={BannerType.success} additionalClassNames="mt-0">
-              <p className="flex justify-center text-base">
-                <CheckmarkCircleOutlineBlackIcon
-                  className="me-3"
-                  mode="success"
-                />
-                <span>
-                  <FtlMsg id="inline-recovery-key-setup-recovery-created">
-                    Account recovery key created
-                  </FtlMsg>
-                </span>
-              </p>
-            </Banner>
+            <BannerNew
+              type="success"
+              content={{
+                localizedHeading: ftlMsgResolver.getMsg(
+                  'inline-recovery-key-setup-recovery-created',
+                  'Account recovery key created'
+                ),
+              }}
+            />
```

Comment:
Much nicer 🧹⭐

I didn't go through and check all of the margin spacing on these fwiw (like `mt-0` being passed in here).